src/snapshot/component_checksum.rs

```diff
@@ -88,7 +88,7 @@ where
 
             trace!(
                 "Component {} has checksum {:X}",
-                bevy::utils::get_short_name(std::any::type_name::<C>()),
+                std::any::type_name::<C>(),
```

Comment:
Yeah, like you said in the pr body, we should figure out what to do here :)

Reply:
Relevant discord conversations: can be found [here](https://discord.com/channels/691052431525675048/1002362493634629796/1314525820877803591)

Reply:
Sages of the discord have said we should use disqualified, since it was created for this. Will do so after I get home.